aiida/restapi/common/utils.py

```diff
@@ -483,6 +483,7 @@ def build_translator_parameters(self, field_list):
         extras = None
         extras_filter = None
         full_type = None
+        profile = None
```

Comment:
`load_profile(None)` defaults to loading the default profile.

I think the correct default here would be the profile that was used in
```verdi -p profile restapi```

Reply:
I have kept this as is, because this is just the parsing of the query parameters and I don't think that should be bothered with determining the default profile. This is now instead handled in `BaseResource.load_profile` which knows what default to use if not specified in the query parameters.